### PR TITLE
Interface tests

### PR DESCRIPTION
**Pull request by realaravinth**
This PR sets up a dev environment for testing, The current testing situation is as follows:

1. `libgit` tests
2. Spins up `interface` in the background: northstar verifies the registrant is indeed an interface instance be hitting `/_ff/interface/versions`(also implemented in this PR)
3. Spins up `northstar` docker container
4. Run tests
5. Tear down env

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -35,14 +35,26 @@ jobs:
...
+        run: virtualenv venv 
+
+      - name: Install dependencies
+        run: make env
+
       - name: generate coverage
         run: make coverage
-
+        env:
+          DYNACONF_GITEA__HOST : "https://git.batsense.net"
```
Comment (dat-adi) [inline .github/workflows/coverage.yml:53]: Wouldn't it be a better approach to have the GITEA_HOST variable present in the dynaconf settings as well?

File: .github/workflows/linux.yml [hunk 2]
```diff
@@ -31,15 +31,25 @@ jobs:
...
+
+      - name: Install dependencies
+        run: make env
 
-      - name: tests
-        run: cd libgit && cargo test --all --all-features --no-fail-fast
+      - name: Run tests
+        run: make test
+        env:
+          DYNACONF_GITEA__HOST : "https://git.batsense.net"
```
Comment (dat-adi) [inline .github/workflows/linux.yml:53]: Similar question to the one posed earlier.

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (realaravinth) [reply .github/workflows/coverage.yml:53]: I didn't want to hard code it in the settings but if the user wants to store the configuration a file, provisions are available for them to use [settings.toml](https://github.com/forgefedv2/interface/blob/master/config/settings.toml) and [.env](https://github.com/forgefedv2/interface/blob/master/config/.env-sample) files

File: .github/workflows/linux.yml (hunk 2 above)
Comment (realaravinth) [reply .github/workflows/linux.yml:53]: answered above https://github.com/forgefedv2/interface/pull/17#discussion_r756642848.

File: interface/runner.py [hunk 3]
```diff
@@ -70,49 +90,48 @@ def get_last_run(self):
...
-                return
-            else:
-                RUNNING = True
-
-            last_run = self.get_last_run()
-            print(last_run)
+        print("running")
+        while True:
+            if self.shutdown_flag.is_set():
+                print("exiting eorker")
```
Comment (dat-adi) [inline interface/runner.py:101]: `eorker` => `worker`.

**Review by dat-adi — COMMENTED** [review]
The clean up is well, *clean*. So, no issues with regards to the changes made in the Makefile or any of the workflows, other than the query about whether strict typing the HOST is what we intend to do.
Tests are approved to proceed further.

File: interface/runner.py (hunk 3 above)
Comment (realaravinth) [reply interface/runner.py]: fixed :rocket:

**Pull request merged**